redbeams/src/main/java/com/sequenceiq/redbeams/flow/redbeams/termination/action/RedbeamsTerminationActions.java

```diff
@@ -99,6 +103,8 @@ protected Selectable createRequest(RedbeamsContext context) {
                 // through the termination
                 metricService.incrementMetricCounter(MetricType.DB_TERMINATION_FINISHED, context.getDBStack());
                 dbStackService.delete(context.getDBStack());
+                dbStackJobService.unschedule(context.getDBStack());
```

Comment:
You should call the `unschedule` at the beginning of the termination. As I see, the termination statuses are not the part of the `getAutoSyncStatuses()`